app/src/components/ViewEventsCards/EventCardElement.tsx

```diff
@@ -124,19 +133,26 @@ export const EventCardElement = ({ eventId, event }: IProps) => {
   );
 };
 
-const colors = [
-  style.cardColorHav,
-  style.cardColorKveld,
-  style.cardColorRegn,
-  style.cardColorSkyfritt,
-  style.cardColorSol,
-  style.cardColorSolnedgang,
-  style.cardColorSoloppgang,
-];
+const colors = (style: any) =>
+  new Map([
+    ['Hav', { style: style.hav, colorCode: hav }],
+    ['Kveld', { style: style.kveld, colorCode: kveld }],
+    ['Regn', { style: style.regn, colorCode: regn }],
+    ['Skyfritt', { style: style.skyfritt, colorCode: skyfritt }],
+    ['Sol', { style: style.sol, colorCode: sol }],
+    ['Solnedgang', { style: style.solnedgang, colorCode: solnedgang }],
+    ['Soloppgang', { style: style.soloppgang, colorCode: soloppgang }],
+  ]);
+
 const getEventHash = (eventId: string): number =>
   [...eventId].map((char) => char.charCodeAt(0)).reduce((a, x) => a + x, 0);
-const getColor = (eventId: string) =>
-  colors[getEventHash(eventId) % colors.length];
+
+export const getColor = (eventId: string, style: any) =>
+  colors(style).get(
+    [...colors(style).keys()][
+      getEventHash(eventId) % [...colors(style).keys()].length
+    ]
```

Comment:
Det blei litt mye kål å ha en Map her, det skal sies 😂 Veit eg sa Map altså, men vi trenger ikkje ha en Map bare for å ha en Map dersom det finnes en vesentlig enklere løsning. Men helt opp til dere!

Reply:
Vi kom frem til map for å kunne lagre de ulike stylene osv inne i samme ting, så vet ikke helt hvordan det kan gjøres enklere.

Reply:
det blir map inntil videre